plugins/plugin-webpack/plugin.js

```diff
@@ -227,14 +227,14 @@ module.exports = function plugin(config, args) {
                     cwd: buildDirectory,
                     configFile: false,
                     babelrc: false,
+                    compact: true,
```

Comment:
Setting `compact` to `false` also works. The default value is `'auto'`, which causes the output from babel. Just FYI.